src/main/java/uk/gov/companieshouse/web/emergencyauthcodeweb/controller/eac/CompanyInformationPageController.java

```diff
@@ -16,7 +16,7 @@
 
 @Controller
 @NextController(ListOfDirectorsController.class)
-@RequestMapping("/request-an-authcode/company/{companyNumber}/company-information")
+@RequestMapping("/auth-code-requests/company/{company_number}/confirm")
 public class CompanyInformationPageController extends BaseController {
```

Comment:
should we rename this controller now?

Reply:
Yeah, you're right, I'll probably have to rename the Officer one as well 👍